alacritty_terminal/src/term/mod.rs

```diff
@@ -360,6 +373,7 @@ impl<T> Term<T> {
             title: None,
             title_stack: Vec::new(),
             selection: None,
+            graphics: Graphics::default(),
```

Comment:
This causes a crash if sixels are drawn without first resizing the window due to the `Graphics.cell_height` and `cell_width` being 0.